frontend/src/components/editor-page/editor-pane/tool-bar/table-picker/table-size-picker-popover.tsx

```diff
@@ -61,7 +61,7 @@ export const TableSizePickerPopover = React.forwardRef<HTMLDivElement, TableSize
                 {...cypressAttribute('col', `${col + 1}`)}
                 {...cypressAttribute('row', `${row + 1}`)}
                 onMouseEnter={onSizeHover(row + 1, col + 1)}
-                title={t('editor.editorToolbar.table.titleWithSize', { cols: col + 1, rows: row + 1 }) ?? undefined}
+                title={t('editor.editorToolbar.table.titleWithSize', { cols: col + 1, rows: row + 1 })}
```

Comment:
Shouldn't this also use `useTranslatedText`?

Reply:
not necessary. the t function is already cached